### PR TITLE
agenda: composer v2 in laminas components instead of using `composer/package-versions-deprecated`

### PR DESCRIPTION
**Pull request by boesing**

### CODE REVIEW

File: meetings/agenda.md [hunk 1]
```diff
@@ -18,3 +18,20 @@ So questions are:
...
+Since then, the usage of `ocramius/package-versions` became obsolete and therefore, `composer/package-versions-deprecated` was already used to "override" `ocramius/package-versions`.
+
+Due to Marcos amazing idea of providing runtime information for composer dependencies, composer added this as a native feature with v2.0.0.
+
+However, we do still depend on the `composer/package-versions-deprecated` dependency as it does support PHP 8.0 and PHP 8.1 in the v1.x range while `ocramius/package-versions` would also require the v2+ version of it.
+In components which do support PHP 7.4 + PHP 8.0, we are forced to use the composer replacement.
+
+We do have two options as of november this year:
+
+1. remove PHP 7.4 support, remove `composer/package-versions-deprecated` and re-add `ocramius/package-versions` since it supports PHP 8.0 as of v2
```
Comment (Ocramius) [inline meetings/agenda.md:34]: This should probably happen anyway, as PHP 7.4 gets EOL'd

File: meetings/agenda.md [hunk 2]
```diff
@@ -18,3 +18,20 @@ So questions are:
...
+
+Due to Marcos amazing idea of providing runtime information for composer dependencies, composer added this as a native feature with v2.0.0.
+
+However, we do still depend on the `composer/package-versions-deprecated` dependency as it does support PHP 8.0 and PHP 8.1 in the v1.x range while `ocramius/package-versions` would also require the v2+ version of it.
+In components which do support PHP 7.4 + PHP 8.0, we are forced to use the composer replacement.
+
+We do have two options as of november this year:
+
+1. remove PHP 7.4 support, remove `composer/package-versions-deprecated` and re-add `ocramius/package-versions` since it supports PHP 8.0 as of v2 
+2. remove `composer/package-versions-deprecated` and require `composer-runtime-api` v2. This will ensure, that we can use `\Composer\InstalledVersions` without the need of any 3rd-party component.
```
Comment (Ocramius) [inline meetings/agenda.md:35]: We should do this too 👍

**Review by Ocramius — APPROVED** [review]

File: meetings/agenda.md (hunk 1 above)
Comment (boesing) [reply meetings/agenda.md:34]: Thats why I mention this. 
When we decide in the next TSC that we are fine with adding composer v2 dependencies, we can immediately drop the composer dependency at all and use composer v2 code.
If we decide to keep support of v1, we have to wait until PHP 7.4 gets EOL'd which is end of november.

File: meetings/agenda.md [hunk 3]
```diff
@@ -18,3 +18,20 @@ So questions are:
...
+
+**Question is:** Do we want to start requiring composer v2 in our components when it comes to features like `\Composer\InstalledVersions` or should we still allow composer v1 even if we started to drop composer v1 in our composer plugins?
```
[the remark this answers is not in the document]
Comment (boesing) [reply meetings/agenda.md:37]: Yah, we do still have versions supporting composer v1 for these use-cases.
So there are no composer v2 only laminas components, there supposed to be at least 1 version with support for composer v1 tho.

File: meetings/agenda.md (hunk 1 above)
Comment (Ocramius) [reply meetings/agenda.md:34]: I see - in general, we don't need to actively work towards it, but rather let renovate figure out whether we can support/what.

Reminds me that I need to start enabling renovate for everything @_@

File: meetings/agenda.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Ocramius) [reply meetings/agenda.md:37]: I don't mind the components bumping to V2. Customers that are still on V1 will first upgrade to older versions of our components.

Declaring a strict supported dependency range will actually ease their upgrade path :+1:

**Pull request merged**